### PR TITLE
Fix bug "Open in Colab" returning "undefined"

### PR DESCRIPTION
**Pull request by theory-in-progress**
### Description

Fix for the Bug which gives the wget link undefined when Opened in Colab

Fix # 

### Additional context

---

### What is the purpose of this pull request?

- [x] Bug fix
- [ ] New feature
- [ ] Other

### CODE REVIEW

File: functions/utils.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export async function pushToGitHub(content, filename, nbUid) {
   })
   try {
     const res = await octokit.request(
-      'PUT /repos/{owner}/{repo}/contents/{path}',
+      'GET /repos/{owner}/{repo}/contents/{path}',
```
Comment (vfdev-5) [inline functions/utils.js:25]: do you still need message and content for get request here ?

File: functions/utils.js [hunk 2]
```diff
@@ -31,8 +31,23 @@ export async function pushToGitHub(content, filename, nbUid) {
...
+          path: `nbs/${nbUid}/${filename}`,
+          message: `nb: add ${nbUid}`,
+          content: content
+        }
+      )
+      return res2.data.content.download_url
+    }
+    else{
+      return `https://raw.githubusercontent.com/pytorch-ignite/nbs/main/nbs/${nbUid}/${filename}`
+    }
```
Comment (vfdev-5) [inline functions/utils.js:49]: Can you change the logic to the following and add comments:
```js
// TODO: add comment
const get_req_result = await octokit.request(
    ...
)

// TODO: add comment
if (get_req_result.status === '200') {
    return ...
}

// TODO: add comment
const put_req_result = await octokit.request(
    ...
)
return put_req_result....
```

File: functions/utils.js (hunk 1 above)
Comment (theory-in-progress) [reply functions/utils.js:25]: Most probably no, I was checking the netlify logs to check what exactly is the download url being returned...

**Review by vfdev-5 — APPROVED** [review]
Thanks for the fix, @theory-in-progress !

**Pull request merged**